### DISCUSSION TITLE
Problems connecting to Qlik Sense Enterprise server

### DISCUSSION THREAD

**Discussion by mountaindude** [q-masters/vsqlik#808] [General]
Background: 

- Connected to corporate network via VPN
- I can connect to the Sense server at https://sense.company.com (not the exact URL, you get the idea..). I.e. port 443
- Logging using regular Windows form based auth
- A reasonably small number of apps on the server (250, maybe?)

When I try to connect using VSQlik nothing happens. 
I then replicated the setup using the same settings, ports etc on a local Sense server, and everything works. 
So the issue seems to be related to the fact that the first server is "far away" from a network perspective.

The question thus: 

**Does VSQlik create any logs that might give hints to what's going on?**
Or does it simply output stuff to the console (available via Help > Developer tools in VSC)?

**Comment by konne** [comment]
Hi Göran,

strange, be honest for the first, I have no idea.
@r-hannuschka please can you describe how Göran can enable the logging?

by
Konrad

**Reply by r-hannuschka** [reply]
Sry for late response, sure i have to check what we currently log not sure it is implemented that deep maybe i have to extend this a bit before.

But the settings should allready exists i will take a look onto this one.

**Comment by r-hannuschka** [comment]
Can you test with version 1.8.0 again, for connection set external which will opens a new Window with electron inside. Maybe we missing some cookies or something else. I will check today about the logging settings too that we also have to extend a bit more.

**Comment by pablocarrillo12** [comment]
I have the same issue and I'm using version 1.8.0
Any advice?

**Reply by wmjekag** [reply]
Same problem. On version 1.8.0.
Even from the qlik server i don´t get anything.

Any advice?
